maintenance-policy.md

```diff
@@ -36,14 +36,11 @@ This file is the canonical source for how the Habitat project is maintained.
 ## Maintainer
 
 * Each component may have multiple Maintainers
-* Handles contributions on GitHub - first response to a PR within 48 hours
+* Handles contributions on GitHub and reviews/responds to PRs
 * Is available on [Slack](http://habitat-sh.slack.com/)
-* Is available to answer mailing list questions within 48 hours
-* Weekends and local holidays in the Maintainer’s jurisdiction are not counted
-  for timeliness requirements. Absences for reasonable causes such as vacations,
-  illness, etc. are also acceptable; Maintainers should notice of absences via
-  slack list whenever possible.
-* Committed to 100% tests passing for your component
+* Is available on the Habitat Forum to answer questions
```

Comment:
I thought we were moving away from any commitment for maintainers to be present in Slack and instead focusing on the forums

Reply:
Good point - I will remove the Slack commitment.